presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionProcess.java

```diff
@@ -264,7 +273,8 @@ public void onFailure(Throwable failedReason)
     private String getProcessWorkingPath(String path)
     {
         File absolutePath = new File(path);
-        File workingDir = new File(SparkFiles.getRootDirectory());
+        String rootDirectory = SparkEnv$.MODULE$.get() != null ? SparkFiles.getRootDirectory() : ".";
```

Comment:
What is this change about? Would you add it to commit message?

Reply:
This change is a safety check mostly for testing scenario where we create the native process object without initial Spark environment (SparkFiles.getRootDirectory will throw NPE in this case), it won't change behavior in normal execution path.

Reply:
It would be nice to add a small comment here to clarify.